src/main/java/seedu/duke/storage/StorageManager.java

```diff
@@ -79,54 +146,93 @@ public void loadAll(Notebook notebook, Timetable timeTable){
+        Scanner s;
+        try {
+            s = new Scanner(f);
+        } catch (FileNotFoundException exception) {
+            throw new SystemException(SystemException.ExceptionType.EXCEPTION_FILE_NOT_FOUND_ERROR);
         }
-        if (!noteDirectory.exists()) {
-            noteDirectory.mkdir();
+
+        while (s.hasNext()) {
+            content.add(s.nextLine());
+        }
+        return content;
+    }
+
+    public static void deleteNoteContentFile(String noteTitle) throws SystemException {
+        String path = FOLDER_DIR + NOTES_DIR + "/" + noteTitle + ".txt";
+        File file = new File(path);
+
+        if (file.exists()) {
+            if (!file.delete()) {
+                throw new SystemException(SystemException.ExceptionType.EXCEPTION_FILE_DELETION_ERROR);
+            }
+        } else {
+            throw new SystemException(SystemException.ExceptionType.EXCEPTION_FILE_NOT_FOUND_ERROR);
         }
     }
 
     /**
-     * Checks if a file exists. If it does not, creates file with the input path
-     * @param path path of file to be created
-     * @throws IOException thrown when directory does not exist. Unable to create file
+     * Saves all the Events in the Timetable to the storage file.
+     *
+     * @param timetable The Timetable containing all the events to be saved.
      */
-    private static void createFile(String path) throws IOException {
-        File file = new File(path);
-        if (!file.exists()) {
-            file.createNewFile();
+    private void saveTimetable(Timetable timetable){
+
+    }
+
+    /**
+     * Saves all the Notes in the Notebook and the Events in the Timetable to the storage file.
+     *
+     * @param notebook The Notebook containing all the notes to be saved.
+     * @param timetable The Timetable containing all the events to be saved.
+     */
+    public void saveAll(Notebook notebook, Timetable timetable)throws SystemException {
+        saveNotebook(notebook);
+        saveTimetable(timetable);
+    }
+
+    /**
+     * Loads the Notebook and Timetable from the storage file.
+     *
+     * @param notebook The Notebook to be loaded into.
+     * @param timetable The Timetable to be loaded into.
+     */
+    public void loadAll(Notebook notebook, Timetable timetable, TagManager tagManager,
+                        StorageManager storageManager) throws SystemException {
+        String path = FOLDER_DIR + NOTEBOOK_FILE_PATH;
+        File f = new File(path);
+        Scanner s;
+        try {
+            s = new Scanner(f);
+        } catch (FileNotFoundException exception) {
+            throw new SystemException(SystemException.ExceptionType.EXCEPTION_FILE_NOT_FOUND_ERROR);
+        }
+        while (s.hasNext()) {
+            String taskDetails = AddNoteCommand.COMMAND_WORD + " " +  s.nextLine();
+            Command command = new Parser().parseCommand(taskDetails);
+            command.setData(notebook, timetable, tagManager, storageManager);
```

Comment:
I see the reason you pass in storageManager is only for setData. Can replace it with 'this' so that this function does not need the StorageManager parameter.

```suggestion
            command.setData(notebook, timetable, tagManager, this);
```